kre-go/context_configuration.go

```diff
@@ -0,0 +1,136 @@
+package kre
+
+import (
+	"fmt"
+
+	"github.com/konstellation-io/kre-runners/kre-go/v4/config"
+	"github.com/konstellation-io/kre/libs/simplelogger"
+	"github.com/nats-io/nats.go"
+)
+
+type Scope string
+
+const (
+	ScopeProject  Scope = "project"
+	ScopeWorkflow Scope = "workflow"
+	ScopeNode     Scope = "node"
```

Comment:
Invert the names to make more sense.

```suggestion
	ProjectScope  Scope = "project"
	WorkflowScope Scope = "workflow"
	NodeScope     Scope = "node"
```

Reply:
Fine by me :ok_hand: